intro-HPC/ch_mympirun.tex

```diff
@@ -0,0 +1,106 @@
+\chapter{Mympirun}
+\label{ch:mympirun}
+
+\lstinline|mympirun| is a tool to make it easier for users of HPC clusters to run
+MPI programs with good performance. We strongly recommend to use \lstinline|mympirun|
+instead of \lstinline|impirun|.
+
+In this chapter, we give a high-level overview. For a more detailed description of all
+options, see the
+\href{https://github.com/hpcugent/vsc-mympirun/blob/master/README.md}{vsc-mympirun README}.
+
+\section{Basic usage}
+\label{sec:myrun-basic-usage}
+
+Before using \lstinline|mympirun|, we first need to load its module:
+
+\begin{prompt}
+%\shellcmd{module load mympirun}%
+\end{prompt}
+
+As an exception, we don't specify a version here. The reason is that we want to ensure
+that the latest version of the \lstinline|mympirun| script is always used, since it may
+include important bug fixes or improvements.
+
+The most basic form of using \lstinline|mympirun| is \lstinline|mympirun [mympirun options] your_program [your_program options]|.
+
+For example, to run a program named \lstinline|example| and give it a single argument (\lstinline|5|),
+we can run it with \lstinline|mympirun example 5|.
+
+
+\section{Controlling number of processes}
+
+There are four options you can choose from to control the number of processes
+\lstinline|mympirun| will start. In the following example, the program \lstinline|mpi_hello|
+prints a single line: \lstinline|Hello world from processor <node> ...| (the sourcecode of
+\lstinline|mpi_hello| is \href{https://github.com/hpcugent/vsc-mympirun/blob/7bb2b70c4db4d780e72cb05014fc505c6f215b26/testscripts/mpi_helloworld.c}{available in the vsc-mympirun repository}).
```

Comment:
just point to `master` rather than a specific commit?

Reply:
I understand the consequences, but I would still rather link to `master`. I think the changes of the file disappearing (or even changing) are quite small.